modules/file-system/gke-storage/storage-class/hyperdisk-ml-sc.yaml.tftpl

```diff
@@ -0,0 +1,24 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ${name}
+  labels:
+  %{~ for key, val in labels ~}
+    ${key}: ${val}
+provisioner: pd.csi.storage.gke.io
+allowVolumeExpansion: true
+parameters:
+  %{~ endfor ~}
```

Comment:
Was this for really meant to be that long?

Reply:
what do you mean by long? FWIW i used the same template as hyperdisk-extreme (noticed its slightly different to hyperdisk-balanced template).

Hyperdisk-ml doesnt [support customizable IOPS so i took that out](https://cloud.google.com/compute/docs/disks/hyperdisks#feature_summary). I also didnt include custom throughput as it automatically specifies this based on [disk size](https://cloud.google.com/compute/docs/disks/hyperdisks#limits-disk)

Reply:
I expected something like this:
```suggestion
  %{~ for key, val in labels ~}
    ${key}: ${val}
  %{~ endfor ~}
provisioner: pd.csi.storage.gke.io
allowVolumeExpansion: true
parameters:
```

I think you don't need to repeat `provsioner` and `allowVolmeExpansion`, and most imporatantly, you don't want them missing if no labels are provided. And to me it looks like `hyperdisk-extreme` has the same issue